docs/blog/unettest.md

```diff
@@ -0,0 +1,288 @@
+# UNETTEST
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When I write software, there are many ways to ask a question. I could go to
+Stack Overflow or a smarter coworker, but I usually like to ask my computer
+first.  What do I do if I feel like my computer is making me wait [seven and a
+half million years](https://hitchhikers.fandom.com/wiki/Deep_Thought) to get an
+answer? 
+
+I would want to revisit the question of how to ask questions. If I want to know
+the Ultimate Answer to Life, the Universe, and Everything, I must ask a damn
```

Comment:
replace "a damn" with "the". Let's keep the language professional even if we don't speak like that in person.